### PR TITLE
Insert debugger breakpoint in the center of the screen

### PR DESCRIPTION
**Pull request by pelikhan**

### CODE REVIEW

File: webapp/src/blocks.tsx [hunk 1]
```diff
@@ -551,6 +551,9 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     insertBreakpoint() {
         if (!this.editor) return;
         const b = this.editor.newBlock(pxtc.TS_DEBUGGER_TYPE);
+        // move block roughly to the center of the screen
+        const m = this.editor.getMetrics();
+        b.moveBy(m.viewWidth / 2, m.viewHeight / 3);
```
Comment (samelhusseini) [inline webapp/src/blocks.tsx]: You can use b.getHeightWidth() to get the size of the block and incorporate that into the calculation if you like

**Comment by pelikhan** [comment]
I'd like to have some kind of pop up animation and close to the menu but that's trickier.

**Comment by samelhusseini** [comment]
Menu as in the flyout? 

Blockly.hideChaff() to close all flyouts, field editors, etc.

**Comment by pelikhan** [comment]
The debugger menu...

**Comment by samelhusseini** [comment]
Ah okay

**Pull request merged**